### DISCUSSION TITLE
Can we support openEuler in future?

### DISCUSSION THREAD

**Discussion by badacook** [cobbler/cobbler#3328] [General]
&#8195; As openEuler has become more and more popular distribution , can we just modifications the signatures.json to add support for openEuler? And in the future can openEuler be supported

**Comment by SchoolGuy** [comment]
Sure you can edit the `signatures.json`. Although it is in an early alpha stage you can try to use https://github.com/cobbler/libcobblersignatures or you can edit the file by hand. After that, you may open a PR to both this repo and https://github.com/cobbler/cobbler.github.io/ to update the signatures for the community as well.
